flow/util/mistune/plugins/tabulate.py

```diff
@@ -0,0 +1,1792 @@
+# -*- coding: utf-8 -*-
```

Comment:
So we're vendoring *tabulate* within the vendored *mistune* package?

Reply:
Thanks for pointing this out, I actually haven't thought about this. I think the benefit of doing this might be that if would be easier for future manage or change if all are directly from our mistune fork and easy to remove if we ever get to merge our branch to mistune. but let me know if there is a strong reason for take tabulate out as a separate verdoring package.

Reply:
No, I think it's fine, I just found it curious. Shouldn't be a licensing issue afaik.